packages/trader/src/Modules/Trading/Components/Form/TradeParams/Duration/duration-mobile.jsx

```diff
@@ -189,16 +192,20 @@ const Numbers = observer(
 
         return (
             <div className='trade-params__amount-keypad'>
-                {show_expiry && (
-                    <Text as='div' size='xxxs' line_height='s' className='expiry-text-container--mobile'>
-                        <Localize
-                            i18n_default_text='Expiry: {{date}}'
-                            values={{
-                                date: !has_error ? setExpiryDate(expiry_epoch, duration_values?.d_duration) : '',
-                            }}
+                <div className='text-container'>
+                    {is_vanilla && (
+                        <DurationRangeText
+                            min={min}
+                            max={max}
+                            duration_unit={getUnitMap()[duration_unit].name_plural}
```

Comment:
@kate-deriv what will we display is `name_plural` is missing in the object?

Reply:
Only in case if duration_unit do not exist in returned object from getUnitMap(), but I got your idea, will add some kind of fallback